### PR TITLE
Bump nbqa

### PR DESCRIPTION
**Pull request by MarcoGorelli**
Made a little patch release to fix simpler of the bugs you reported (the other one will take a bit longer I'm afraid)

BTW if you enable pre-commit-ci on https://results.pre-commit.ci/ then the pre-commit checks will run automatically on each PR, it's pretty neat

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -5,13 +5,13 @@ repos:
...
 - repo: https://github.com/nbQA-dev/nbQA
-  rev: 0.4.0
+  rev: 0.4.1
   hooks:
   - id: nbqa-black
     additional_dependencies: [black==20.8b1]
     args:
       - "--nbqa-mutate"
-    exclude: 00-intro.ipynb|05-pybind.ipynb
+    exclude: 05-pybind\.ipynb$
```
Comment (MarcoGorelli) [inline .pre-commit-config.yaml:14]: this takes a regular expression, so `.` will match anything (rather than a literal dot)

Comment (henryiii) [reply .pre-commit-config.yaml:14]: Ah, yes, you are right. 🤦

**Comment by henryiii** [comment]
I tried to add results.pre-commit on CLI11, but ran into the problem that Docker is not supported - see https://github.com/pre-commit-ci/issues/issues/11 . It should work for non-C++ repos, though, like this one.

**Comment by henryiii** [comment]
Fantastic, thanks for addressing this!

**Comment by henryiii** [comment]
9 minutes before the start of class, too. ;)

**Pull request merged**